content/en/blog/_posts/2025-02-21-introducing-gateway-api-inference-extension/index.md

```diff
@@ -92,6 +97,11 @@ As the Gateway API Inference Extension heads toward GA, planned features include
 
 ## Summary
 
-By aligning model serving with Kubernetes-native tooling, Gateway API Inference Extension aims to simplify and standardize
-how AI/ML traffic is routed. With model-aware routing, criticality-based prioritization, and more, it helps ops teams deliver
-the right LLM services to the right users—smoothly and efficiently.
+By aligning model serving with Kubernetes-native tooling, Gateway API Inference Extension aims to simplify
+and standardize how AI/ML traffic is routed. With model-aware routing, criticality-based prioritization, and
+more, it helps ops teams deliver the right LLM services to the right users—smoothly and efficiently.
+
+**Ready to learn more?** Visit the [project docs](https://gateway-api-inference-extension.sigs.k8s.io/) to dive deeper,
+give Inference Extension a try with a few [simple steps](https://gateway-api-inference-extension.sigs.k8s.io/guides/),
```

Comment:
I would suggest saying "... give the Inference Gateway extension a try with a few ...".

Reply:
We probably want to hold off on publishing this until we've updated our guides to use proper "Inference Gateways" instead of Envoy patches. Maybe that's actually an argument for saving this until after KubeCon?

Reply:
The initial inference extension supported landed in kgateway and I plan on adding an inference extension docs PR in the next few days.